var/spack/repos/builtin/packages/fenics-dolfinx/package.py

```diff
@@ -58,7 +71,7 @@ class FenicsDolfinx(CMakePackage):
     variant("slepc", default=True, description="slepc support")
```

Comment:
Should `slepc` be "on" by default?

Reply:
I think on by default is ok.

Reply:
I'm not sure I agree @garth-wells. I would be more inclined to make the defaults as minimal as possible.